content/webapp/pages/guides.tsx

```diff
@@ -0,0 +1,131 @@
+import { NextPageContext } from 'next';
+import { FunctionComponent, ReactElement } from 'react';
+import PageLayout from '@weco/common/views/components/PageLayoutDeprecated/PageLayoutDeprecated';
+import SpacingSection from '@weco/common/views/components/SpacingSection/SpacingSection';
+import LayoutPaginatedResults from '@weco/common/views/components/LayoutPaginatedResults/LayoutPaginatedResults';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
+import SegmentedControl from '@weco/common/views/components/SegmentedControl/SegmentedControl';
+import { getGuides } from '@weco/common/services/prismic/guides';
+import { getGuideFormatId } from '@weco/common/services/prismic/utils';
+import {
+  GuideFormatId,
+  GuideFormatIds,
+} from '@weco/common/model/content-format-id';
+import { Page } from '@weco/common/model/pages';
+import { PaginatedResults } from '@weco/common/services/prismic/types';
+
+const pageDescription = 'Guides intro text...';
+const displayTitle = 'Guides';
+
+type FiltersProps = {
+  currentId: GuideFormatId;
+};
+
+const Filters: FunctionComponent<FiltersProps> = ({ currentId }) => {
+  return (
+    <Layout12>
+      <SegmentedControl
+        id={'guidesFilter'}
+        activeId={currentId || 'all'}
+        items={[
+          {
+            id: 'all',
+            url: '/guides',
+            text: 'All',
+          },
+          {
+            id: GuideFormatIds.HowTo,
+            url: '/guides?format=how-to',
```

Comment:
These keys are not in any CMS system, so will be very hard to maintain as we would ahve to keep them in the frontend code. Could we just use IDs?